src/MeshManager.cpp

```diff
@@ -46,73 +95,20 @@ namespace blitzdg {
     /**
      * Read a .msh file that was generated with Gmsh.
      */
-    void MeshManager::readMesh(string gmshInputFile) {
+    void MeshManager::readMesh(const string& gmshInputFile) {
```

Comment:
By making this a reference, do we avoid possibly copying strings all over the place?

Reply:
Yup. No copies made here.